service-manual/making-software/information-security.md

```diff
@@ -26,15 +26,14 @@ taken to guarantee its security.
 
 Information security is a topic both broad and deep, drawing from fields ranging from economics and psychology through to mathematics and probability.
 
-This document doesn’t provide a thorough review of the field. Instead, it aims to:
+This document doesn't provide a thorough review of the field. Instead, it aims to:
```

Comment:
Yes, that's my editor.
I thought the rule was to use ' and let the markdown processor build out the appropriate quotes.

Happy to change if that's wrong